src/types.ts

```diff
@@ -1793,13 +1793,17 @@ export type Sort<T> = {
   [P in keyof T]?: AscDesc;
 };
 
+export type SortMember<T> = {
```

Comment:
Why this type contains Member in its name? Seems like generic that just differs from Sort in that it allows string[]. What string values can be passed here?

Reply:
I'm adding support to sort members by role, and the idea is to pass something like {channel_role:["channel_admin", "channel_member"]}

Reply:
👍